Common/Data/Text/Parsers.cpp

```diff
@@ -63,7 +63,7 @@ bool ParseMacAddress(std::string str, uint8_t macAddr[6]) {
 	return true;
 }
 
-bool TryParse(const std::string &str, uint32_t *const output) {
+static bool ParseUnsigned32(const std::string &str, uint32_t *const output) {
```

Comment:
TryParseUnsigned32?

(I find it nice to be clear about whether functions like this can fail or not, based on experience with a certain standard library function...)